packages/react-native/android/src/main/java/backtrace/library/BacktraceDatabase.java

```diff
@@ -0,0 +1,12 @@
+package backtraceio.library;
+
+public class BacktraceDatabase {
```

Comment:
Can we rename it to something else? RIght now, I'm not sure if this is the same database that we have in the JS layer or not.

Reply:
Unfortunately no, without changes in our NDK library. Unfortunately We need to stick to class names available in the BacktraceAndroid. React always prefer to add a library prefix and NDK cannot detect exports without the library namespace